lib/ok.ex

```diff
@@ -116,6 +116,40 @@ defmodule OK do
 
   def check({:error, reason}, _func, _reason), do: {:error, reason}
 
+  @doc """
+  Helper function. Checks if a result tuple is tagged as `:ok`, and returns `true` if so.
```

Comment:
Don't bother with `Helper function` (my opinion is they're all helpful) :-)

Reply:
Done ;)